_maps/configs/nemo-class.json

```diff
@@ -1,19 +1,30 @@
 {
-	"map_name": "Nemo-class Spacelife Research Vessel",
+	"map_name": "Nemo-class Fauna Research Vessel",
 	"map_short_name": "Nemo-class",
 	"map_path": "_maps/shuttles/shiptest/nemo-class.dmm",
 	"map_id": "nemo-class",
 	"job_slots": {
```

Comment:
At the moment, every job uses the default (Nanotrasen) outfits. Since this is an independent vessel, you may wish to re-flavor them with alternate outfits.

Reply:
I don't think there's independent outfits for all the jobs available though. It seems a little inconsistent to have some that are styled like NT and some that are independent. Should I still switch the available ones to independent or just keep them all the same for consistency?

Reply:
Changed it. Do the job titles and outfits seem appropriate?

Reply:
The new titles and outfits all look good